### PR TITLE
Make Gantt tooltip the same as Tree and Graph view

### LINKED ISSUE #8210: UI time zone not used in Gantt charts tooltips
**Apache Airflow version**: v2.0.0.dev0

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): -

**Environment**: Docker 

- **Cloud provider or hardware configuration**: -
- **OS** (e.g. from /etc/os-release): Debian GNU/Linux 10 (buster)
- **Kernel** (e.g. `uname -a`): 
Linux 6dac905dd519 4.19.76-linuxkit #1 SMP Thu Oct 17 19:31:58 UTC 2019 x86_64 GNU/Linux
- **Install tools**: Airflow Breeze (image: apache/airflow:master-python3.6-ci)
- **Others**: -
**What happened**:

Gantt charts seem to always use UTC for displaying task start/end dates, but use local time for time axis, regardless of the time zone set in UI (see screens below: my local time zone is CEST).
Zrzut ekranu 2020-04-8 o 09 18 50
Zrzut ekranu 2020-04-8 o 09 17 17
Zrzut ekranu 2020-04-8 o 10 58 09

**What you expected to happen**:

Time zone of the time axis and task start/end dates should be the same - probably the time zone specified in the UI should be used.

**How to reproduce it**:

Simply go to Gantt chart of any Dag Run and try changing the time zone in the UI to see that neither the time axis nor task start/end dates reflect the change.

You can also change the local time on your workstation and refresh the page with Gantt chart to see that time axis changed to reflect the new time zone.

**Anything else we need to know**: -

### PR DESCRIPTION
**Pull request by ashb**
To make this work I needed to change the fields exposed via the json
representation to those expected via the `tiTooltip` function.

Closes #8210.

**Before**

image

**After**:

image

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by ashb** [comment]
Though it loos like there is a bug in the Duration display for the tiTooltip function I reused.

File: airflow/www/static/js/gantt-chart-d3v2.js [hunk 1]
```diff
@@ -118,18 +106,17 @@ d3.gantt = function() {
...
   var taskTypes = [];
-  var taskStatus = [];
   var height = document.body.clientHeight - margin.top - margin.bottom-5;
   var width = $('.gantt').width() - margin.right - margin.left-5;
 
   var tickFormat = "%H:%M";
 
   var keyFunction = function(d) {
-    return d.startDate + d.taskName + d.endDate;
+    return d.start_ate + d.task_id + d.end_date;
```
Comment (kaxil) [inline airflow/www/static/js/gantt-chart-d3v2.js]: Typo

Comment (kaxil) [inline airflow/www/static/js/gantt-chart-d3v2.js]:
```suggestion
    return d.start_date + d.task_id + d.end_date;
```

**Review by kaxil — APPROVED** [review]
1 Typo that needs fixing or else LGTM

File: airflow/www/static/js/gantt-chart-d3v2.js (hunk 1 above)
Comment (ashb) [reply airflow/www/static/js/gantt-chart-d3v2.js]: Whops :)

**Pull request merged**